src/test/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+uk.gov.companieshouse.scanupondemand.orders.api.health=/orderable/scan-upon-demand/healthcheck
+uk.gov.companieshouse.scanupondemand.orders.api.home=/orderable/scan-upon-demand
+
+spring.data.mongodb.uri: ${MONGODB_URL}
```

Comment:
I think `spring.data.mongodb.uri: ${MONGODB_URL}` may need to be moved from this properties file to the application's real properties file? For me, the presence of this property here causes `java.lang.IllegalStateException: Failed to load ApplicationContext` in some tests.